live-build/misc/ansible-roles/appliance-build.crash-python-development/tasks/main.yml

```diff
@@ -19,6 +19,15 @@
     name:
       - git
       - python3-pyelftools
+      - python3-pip
+    state: present
+  register: result
+  until: result is not failed
+  retries: 3
+  delay: 60
+
+- pip:
```

Comment:
Do we need to ensure we use “pip3”? Can you verify this will install for python3 on the image you tested?

Reply:
It uses pip3 by default because we set `ansible_python_interpreter` to python3. I have verified that the mypy installed is based on python3.